app/controllers/users_controller.rb

```diff
@@ -10,15 +10,15 @@ def show
         render :json => users
     end
 
-    def user_Todos
+    def user_todos
         userId = User.find_by(id: current_user.id)
         todos=Todo.where(id:userId)
         render :json => todos
     end
 
-    def update_Email
-        duplicated_Email=User.find_by(email: user_params[:email])
-        if duplicated_Email==nil
+    def update_email
+        user_duplicated_email=User.find_by(email: user_params[:email])
+        if user_duplicated_email==nil
```

Comment:
これだとまだ重複した`email`であるように思います🤔
`duplicated`をもし使用したいのであれば、`duplicated_user`十分伝わります
(理由)
- 取得しているときの = の右辺を見れば`email`で取得していることがわかる
- この変数を使われは重複したユーザがいるかどうかなのでこれ以降の行ではemailがどうこうという情報はいらない

↑ 上記で色々言ってますが、下記のようにすれば気にしなくてすむので、あくまで考え方の参考に笑

```suggestion
        unless User.exists?(email: user_params[:email])
```